routes/lang/en.json

```diff
@@ -417,10 +417,10 @@
   "app.versionexists": "You already have this version of the plugin installed.",
   "app.unknownuser": "Unknown User",
   "app.savetemplate": "Save Template",
-  "app.deleteTempelate": "Delete %{type}",
-  "app.confirmdeletetemplate": "Are you sure you want to delete this %{type}?",
-  "app.confirmdeletetemplatewarning": "<b>IMPORTANT</b>: This will delete all the information in this %{type}.",
-  "app.addtemplate": "Add Template",
+  "app.deleteTempelate": "Delete Template",
+  "app.confirmdeletetemplate": "Are you sure you want to delete this Template?",
+  "app.confirmdeletetemplatewarning": "<b>IMPORTANT</b>: This will delete all the information in this Template.",
```

Comment:
@karthickrk1 - can we check the text from the word document Poornima provided.
here and other-places also